charm/kafka/reactive/tls.py

```diff
@@ -163,3 +163,63 @@ def import_ca_crt_to_keystore():
 
             remove_state('tls_client.ca_installed')
             set_state('kafka.ca.keystore.saved')
+
+
+@when('endpoint.certificates.departed')
+def clear_certificates():
+    ca_path = '/usr/local/share/ca-certificates/{}.crt'.format(
+        hookenv.service_name()
+    )
+    ca_keystore = os.path.join(
+        KAFKA_SNAP_DATA,
+        "kafka.server.truststore.jks"
+    )
+    if os.path.exists(ca_path):
+        os.remove(ca_path)
+    if os.path.exists(ca_keystore):
+        os.remove(ca_keystore)
+
+    certs_paths = unitdata.kv().get('layer.tls-client.cert-paths', {})
+    for cert_type in ('server', 'client'):
+        crt_path = os.path.join(
+            KAFKA_SNAP_DATA,
+            "{}.crt".format(cert_type)
+        )
```

Comment:
Might turn all these path-resolving snippets into functions or staticmethods as well, so that all the handlers are consistent.

Reply:
done.